lib/chef/monkey_patches/net-ssh-multi.rb

```diff
@@ -54,6 +54,39 @@ def busy?(include_invisible=false)
         end
 
         class Session
+          # ===== PATCH START
+          # This is used in :retry_times :on_error strategy in next_session() below
+          # and defines how many times connection will be retried before deemed as failed
+          #
+          # Helps with an unknown issue on OSX (at least Yosemite, but observed on earlier versions
+          # where sockets can't be open from the first attempt to hosts which are alive and well.
+          # E.g. the socket(2) call in Net::SSH returns bogus error like "No route to host"
+          #
+          # Since @retries_max needs to be passed for :retry_times, new behavior of :on_error
+          # attr_accessor is needed and initializer need to be monkey-patched to include
+          # two instance vars into it
+          attr_accessor :retries_max
+
+          def initialize(options={})
```

Comment:
Is it possible to `super` here and only add the custom behavior we need instead of copying the whole method body?